app/models/comment.rb

```diff
@@ -46,14 +48,17 @@ class Comment < ActiveRecord::Base
     participation.unparticipate! if participation.present?
   end
 
-  def message
-    @message ||= Diaspora::MessageRenderer.new text
-  end
-
   def text= text
      self[:text] = text.to_s.strip #to_s if for nil, for whatever reason
   end
 
+  def people_allowed_to_be_mentioned
+    return @people_allowed_to_be_mentioned unless @people_allowed_to_be_mentioned.nil?
+    contacts = author.owner.try(:contact_people) if parent.public?
+
+    @people_allowed_to_be_mentioned = parent.try(:participants).to_a.concat(contacts.to_a).push(parent.author).uniq
```

Comment:
>Am I right that we don't have separate photos comments allowed now?

At the moment, the interface doesn't allow users to comment on a specific photo, they can only comment on the post which contains the photo. But I guess this can change in the future.